src/exercises/compile-and-run/README.md

```diff
@@ -1,30 +1,31 @@
-# Compile and run RISC-V and CHERI-RISC-V programs
+# Compile and run baseline and CHERI-enabled programs
 
 This exercise steps you through getting up and running with code compilation
-and execution for RISC-V and CHERI-RISC-V programs.
+and execution for baseline and CHERI-enabled programs.
 
 The first test program is written in conventional C, and can be compiled
-to RISC-V or CHERI-RISC-V targets:
+to RISC-V, ARMv8, CHERI-RISC-V or Morello targets:
 
```

Comment:
I wonder, if, throughout, you should adopt the parlance ‘baseline’ and ‘CHERI-enabled’ rather than trying to name check all the architectural combinations, which gets a bit awkward. And is not necessarily beneficial. If you feel that’s not specific enough, you could have some text up here making it clear what those terms refer to.

Reply:
@gvnn3 has done mostly that; I'll touch up any residual places I find.